build/external-dependencies.props

```diff
@@ -354,11 +353,6 @@
     <ExternalDependency Include="StackExchange.Redis.StrongName" Version="$(StackExchangeRedisStrongNamePackageVersion)" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="StreamJsonRpc" Version="$(StreamJsonRpcPackageVersion)" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="StyleCop.Analyzers" Version="$(StyleCopAnalyzersPackageVersion)" Source="$(DefaultNuGetFeed)" Private="true" />
-    <ExternalDependency Include="System.Data.SqlClient" Version="$(Benchmarks_11_SystemDataSqlClientPackageVersion)" Source="$(DefaultNuGetFeed)" Private="true">
-      <!-- This version is used by EF's 1.1 benchmark project -->
-      <NoWarn>KRB2004</NoWarn>
```

Comment:
Can you also remove the NoWarn on the other System.Data.SqlClient package in this list? Should be unnecessary now.

Reply:
Done